### PR TITLE
Dropped support for python 3.5

### PR DESCRIPTION
**Pull request by mraarif**

### CODE REVIEW

**Review by UsamaSadiq — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+	pip-compile --upgrade --allow-unsafe --rebuild -o requirements/pip.txt requirements/pip.in
 	pip-compile --upgrade -o requirements/pip_tools.txt requirements/pip_tools.in
 	pip-compile --upgrade -o requirements/base.txt requirements/base.in
 	pip-compile --upgrade -o requirements/docs.txt requirements/docs.in
 	pip-compile --upgrade -o requirements/test.txt requirements/test.in
 	pip-compile --upgrade -o requirements/tox.txt requirements/tox.in
-	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
+	pip-compile --upgrade -o requirements/ci.txt requirements/ci.in
 	# Let tox control the Django version for tests
-	grep -e "^django==" requirements/base.txt > requirements/django.txt
```
Comment (jmbowman) [inline Makefile:11]: Don't we still want the `django.txt` file created here?

File: requirements/pip.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    make upgrade
+#
```
Comment (jmbowman) [inline requirements/pip.txt:6]: `pip.in` is empty, so this one is essentially also; don't you want a few things like `pip`, `setuptools`, and `wheel` in there?

File: tox.ini [hunk 3]
```diff
@@ -1,10 +1,12 @@
 [tox]
-envlist = {py35,py38}-django{22}
+envlist = py38-django{22,30,31}
 
 [testenv]
 deps =
-    django22: -r requirements/django.txt
-    -r{toxinidir}/requirements/test.txt
+    django22: Django>=2.2,<2.3
```
Comment (jmbowman) [inline tox.ini]: Here also I thought we'd want to keep `django.txt` to be explicit about which 2.2.x version we're testing.

File: wiki/models/__init__.py [hunk 4]
```diff
@@ -1,6 +1,3 @@
-# -*- coding: utf-8 -*-
-from __future__ import absolute_import
-
 from django.conf import settings as django_settings
 from django.core.exceptions import ImproperlyConfigured
 from six import string_types, text_type
```
Comment (jmbowman) [inline wiki/models/__init__.py]: Aren't these imports from `six` now unused?

**Comment by UsamaSadiq** [comment]
The edx-platform PR https://github.com/edx/edx-platform/pull/26085 made from this fork has all tests passing so after fixing up the suggestions, this PR can be merged.

File: Makefile (hunk 1 above)
Comment (UsamaSadiq) [reply Makefile:11]: reverted the change as per suggestion.

File: requirements/pip.txt (hunk 2 above)
Comment (UsamaSadiq) [reply requirements/pip.txt:6]: Updated the `pip.in` file with `pip`, `setuptools` and `wheel` values.

File: tox.ini (hunk 3 above)
Comment (UsamaSadiq) [reply tox.ini]: reverted the `django.txt` change.

File: wiki/models/__init__.py (hunk 4 above)
Comment (UsamaSadiq) [reply wiki/models/__init__.py]: Removed the unused statement.

**Review by awais786 — APPROVED** [review]

**Pull request merged**